lib/vue-simple-suggest.vue

```diff
@@ -266,18 +280,16 @@ export default {
       this.text = text
     },
     select (item) {
-      this.hovered = null
       this.selected = item
 
       this.$emit('select', item)
-
+      this.hover(null)
```

Comment:
Why do we need to emit a `hover` event on `select`?

Reply:
This may be left to be `this.hovered = null`.

Reply:
Ok, I see now, thanks for a comprehensive reply. It makes sense to emit an event in that case.

The idea about taking away the pain has the very same intent we had when writing this component. So, I'd say, it's right in the spot.
We'd be very glad, if you implemented this.

Reply:
I will try this approach then.